test/inverse_of_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+class InverseOfTest < IdentityCache::TestCase
+  def setup
+    super
+
+    @item = Item.create!(:title => 'bob')
+
+    AssociatedRecord.create!(:item => @item)
+    AssociatedRecord.create!(:item => @item)
+    AssociatedRecord.create!(:item => @item)
+  end
+
+  def test_fetch_should_setup_association_for_cache_belongs_to
+    Item.cache_belongs_to :associated
```

Comment:
This seems like it should raise, since the model doesn't even have a `belongs_to :associated` association, nor does the table have an associated_id column on the items table.  I think this must be actually using the has_one association.